content/en/docs/concepts/security/index.md

```diff
@@ -541,13 +538,19 @@ authorization policies using `.yaml` files.
 
 ### Implicit enablement
 
-You don't need to explicitly enable Istio's authorization features. Just apply
-an authorization policy to the workloads to enforce access control.
-For workloads without authorization policies applied, Istio doesn't enforce
-access control allowing all requests.
+You don't need to explicitly enable Istio's authorization features; they are available after installation.
+To enforce access control to your workloads, you apply an authorization policy.
 
-Authorization policies support `ALLOW`, `DENY` and `CUSTOM` actions. The policy precedence is
-`CUSTOM`, `DENY` and `ALLOW`. The following graph shows the policy precedence in detail:
+For workloads without authorization policies applied, Istio allows all requests.
+
+Authorization policies support `CUSTOM`, `DENY` and `ALLOW` actions. You can apply multiple policies, each with a
+different action, as needed to secure access to your workloads.
```

Comment:
Many fewer people will use CUSTOM, right?  I'd stay with ALLOW, DENY and CUSTOM in that order

Reply:
updated.